frontend/src/components/CardOrder/CardOrder.jsx

```diff
@@ -0,0 +1,74 @@
+import './CardOrder.scss'
+
+import { useSelector } from 'react-redux'
+import { adminSelectors } from '../../store/admin/adminSelectors'
+
+import AdditionalInfoAdmin from '../AdditionalInfoAdmin/AdditionalinfoAdmin'
+
+function CardOrder() {
+  const viewTab = useSelector(adminSelectors.getAdminTab)
+  const viewExtra = useSelector(adminSelectors.getStatusExtra)
+  console.log(viewExtra)
+  return (
+    <div className="card-order grid">
+      <p className="grid__item text-s">
+        23.09.2023
+        <br />
+        09:15
+      </p>
+      <p className="grid__item text-m-bold">#145639000</p>
+      <p className="grid__item text-m-bold">Генеральная</p>
+      <p className="grid__item text-m-bold">
+        23.09.2023
+        <br />
+        12:00 - 15:00
+      </p>
+      <p className="grid__item text-m-bold">5900</p>
+      <div className="grid__item text-m-bold card-order__complete">
+        {viewTab === 'new' ? (
+          <>
+            <button className="card-order__button text-m-bold">Принять</button>
+            <div className="card-order__close">+</div>
+          </>
+        ) : (
+          ''
+        )}
```

Comment:
Лучше использовать оператор && в условном рендеринге или возвращать null вместо пустой строки
```suggestion
                {viewTab === 'new' && (
          <>
            <button className="card-order__button text-m-bold">Принять</button>
            <div className="card-order__close">+</div>
          </>
        )}
```